api/jobs.py

```diff
@@ -231,6 +251,20 @@ def next(self):
         if result == None:
             self.abort(400, 'No jobs to process')
 
+        # Second, update document to store formula request.
+        result = self.app.db.jobs.find_one_and_update(
+            {
+                '_id': result['_id']
+            },
+            { '$set': {
+                'request': generate_formula(result['intent'])}
+            },
+            return_document=pymongo.collection.ReturnDocument.AFTER
+        )
+
+        if result == None:
```

Comment:
Type comparison makes sense. I want the explicit nil check, so I'll change to `is None` and merge.